scalastyle-config.xml

```diff
@@ -0,0 +1,79 @@
+<scalastyle>
+  <name>Scalastyle standard configuration</name>
+
+  <check level="error" enabled="true" class="org.scalastyle.file.FileTabChecker"/>
+  <check level="error" enabled="true" class="org.scalastyle.file.WhitespaceEndOfLineChecker"/>
+  <check level="error" enabled="true" class="org.scalastyle.file.NewLineAtEofChecker"/>
+
+  <!--Indentation checker, whilst useful gives false positives on the file headers, so disabled for now-->
+  <check level="error" enabled="false" class="org.scalastyle.file.IndentationChecker"/>
+
+  <check level="error" enabled="true" class="org.scalastyle.file.FileLineLengthChecker">
+    <parameters>
+      <parameter name="maxLineLength"><![CDATA[120]]></parameter>
```

Comment:
It is our *hard* limit. The soft limit is 80, but I there's no way I can teach ScalaStyle about what a soft limit is ^^